src/styles/item/_persistent-damage-editor.scss

```diff
@@ -1,36 +1,30 @@
-.persistent-damage-dialog {
-    /** Work around Font Awesome styling bug present as of version 6.2 */
-    i.fa-fw {
-        min-width: 1em;
-    }
-
-    h3 {
+.persistent-damage-editor .window-content {
+    header {
         display: flex;
         align-items: baseline;
         justify-content: space-between;
-        a {
-            float: right;
-            font-size: 0.8em;
-            &:hover i {
-                @include die-spin;
-            }
+        white-space: nowrap;
+        font-size: var(--font-h6-size);
+        margin-bottom: var(--space-4);
+        a :hover i {
+            @include die-spin;
         }
```

Comment:
Use buttons here (className of `inline-control icon fa-solid fa-...`)

Reply:
Done. The die spin looks awful there so I removed it.